eclass/qt5-build.eclass

```diff
@@ -125,14 +125,21 @@ fi
 
 LICENSE="|| ( GPL-2 GPL-3 LGPL-3 ) FDL-1.3"
 
-case ${PN} in
-	assistant|linguist|qdbus|qdbusviewer|pixeltool)
-		SLOT=0 ;;
-	linguist-tools|qtdiag|qtgraphicaleffects|qtimageformats|qtpaths| \
-	qtplugininfo|qtquickcontrols|qttranslations|qtwaylandscanner)
-		SLOT=5 ;;
+case ${PV} in
```

Comment:
a small formality: can you squash this commit with the earlier commit titled "qt5-build.eclass: Change SLOTing" since it's just a fixup?

Reply:
sure, that was the plan ;)